player/public/js/content-renderer.js

```diff
@@ -332,7 +332,8 @@ org.ekstep.contentrenderer.web = function (id) {
 
 org.ekstep.contentrenderer.device = function () {
 	var globalconfig = EkstepRendererAPI.getGlobalConfig()
-	if (isMobile) {
+	var isMobile = (/android|webos|iphone|ipad|ipod|blackberry|iemobile|opera mini/i.test(navigator.userAgent.toLowerCase()));
```

Comment:
Why are you adding this new logic to identify mobile now. If you still need mobile check, go with window.cordova itself.